src/electron/utils.ts

```diff
@@ -16,11 +16,12 @@ export function isMacAppStore() {
 
 export function isWindowsStore() {
     const isWindows = process.platform === 'win32';
-    if (isWindows && !process.windowsStore &&
+    let windowsStore = process.windowsStore;
+    if (isWindows && !windowsStore &&
         process.resourcesPath.indexOf('8bitSolutionsLLC.bitwardendesktop_') > -1) {
-        process.windowsStore = true;
+        windowsStore = true;
     }
-    return isWindows && process.windowsStore && process.windowsStore === true;
+    return isWindows && windowsStore && windowsStore === true;
```

Comment:
Can't this be simplified down to `return isWindows && windowsStore === true;` ?

Reply:
is this nitpick the last thing holding off native possible Apple Silicon support?

Reply:
Hi @flying-sausages, no, this is not holding anything up, we just were waiting for our prior release before approving/merging this in. This nit was just general feedback.

Reply:
Sorry, my phrasing wasn't the most appropriate! Thanks for the clarification :)

Reply:
@cscharf Probably, can't see any case where this simplification would not work.